### PR TITLE
Feedback Sign-In Fix, TeamInfo & LH Updates

### PR DESCRIPTION
**Pull request by amolina38**
-fixed sign-in/sign-out visibility bug on feedback page
-added section for contributions to team info page
-added peer review survey
-fixed build error
-improved team info page LH scores

### CODE REVIEW

File: .env
Comment (ajbieber) [inline .env]: Make sure to remove this. Once Nasa's PR is merged, the gitignore should block this from being merged in the future

**Review by ajbieber — CHANGES_REQUESTED** [review]

File: .env
Comment (amolina38) [reply .env]: deleted file

**Review by ajbieber — APPROVED** [review]

**Pull request merged**
